components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -75,8 +75,8 @@ export class TeamDBImpl implements TeamDB {
         if (!name) {
             throw new Error('Team name cannot be empty');
         }
-        if (!/^[A-Za-z0-9 _-]+$/.test(name)) {
-            throw new Error('Please choose a team name containing only letters, numbers, -, _, or spaces.');
+        if (!/^[A-Za-z0-9 '_-]+$/.test(name)) {
```

Comment:
👍🏻 that should work well with URI components

Reply:
Ah, actually I also wanted to replace them with `-` in the `slug` below (thanks for the reminder). I think keeping `'` in URLs is almost like asking for trouble (even if it's URL-safe).